OSSM-8570: 2.6.5 (At Stage): [DOC] Release Notes (#87678)
openshift/openshift-docs: modules/ossm-release-2-6-5.adoc

////
Module included in the following assemblies:
* service_mesh/v2x/servicemesh-release-notes.adoc
////

:_mod-docs-content-type: REFERENCE
[id="ossm-release-2-6-5_{context}"]
= {SMProductName} version 2.6.5

This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.5, and includes the following `ServiceMeshControlPlane` resource version updates: 2.6.5, 2.5.8, and 2.4.14.

This release addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.14 and later.

include::snippets/ossm-current-version-support-snippet.adoc[]

> The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource. The version of {SMProductShortName} automatically ensures a compatible version of Kiali.

Reviewer: ```suggestion
You can use the most current version of the {KialiProduct} with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource. The version of {SMProductShortName} automatically ensures a compatible version of Kiali.
```

here an active voice can be used.

[id=ossm-release-2-6-5-components_{context}]
== Component updates

|===
|Component |Version

|Istio
|1.20.8

|Envoy Proxy
|1.28.7

|Kiali Server
|1.73.18
|===

[id="ossm-new-features-2-6-5_{context}"]
== New features

* {TempoName} Stack is now supported on {ibm-z-title}.

[id="ossm-fixed-issues-2-6-5_{context}"]
== Fixed issues

> * https://issues.redhat.com/browse/OSSM-8608[OSSM-8608] Previously, terminating a CNI pod during the installation phase while copying binaries could leave Istio-CNI temporary files on the node's file system. Repeated occurrences could eventually fill up the node's disk space. Now, while terminating a CNI pod during the installation phase firstly deletes existing temporary files before copying the CNI binary, ensuring that only one temporary file per Istio version exists on the node's file system.

Reviewer: Same comment as earlier
